src/vue/forms/MarketOrders/CreateTradeOfferForm.vue

```diff
@@ -143,6 +156,25 @@ export default {
     config,
     isOfferCreating: false,
   }),
+  computed: {
+    ...mapGetters([
+      vuexTypes.accountBalances,
+    ]),
+    baseAssetBalance () {
+      return (this.accountBalances
+        .find(i => i.asset === this.assetPair.base) || {}).balance
+    },
+    quoteAssetBalance () {
+      return (this.accountBalances
+        .find(i => i.asset === this.assetPair.quote) || {}).balance
+    },
+    formQuoteAmount () {
+      return MathUtil.multiply(this.form.price, this.form.amount)
+    },
+    totalValue () {
+      return +this.formQuoteAmount ? this.formQuoteAmount : ''
+    },
```

Comment:
can we merge its logic with `formQuoteAmount`?

Reply:
We can but it will not be entirely right, as we will duplicate the logic. Because this property is used in the components.